packages/transaction-manager/specs/encryption.md

```diff
@@ -19,6 +19,23 @@ A "normalized Keccak256 hash" of a JSON object is Keccak256 hash of an object wh
 - the properties have been sorted alphabetically
 - all the values and properties have been lowercased
 
+## Note on multiformatted identities
```

Comment:
should this be in encryption or request logic?

Reply:
Encryption because here we refer to encryption identities = encryption key pairs (cf. the above paragraph)